es6/002_variable/variable.js

```diff
@@ -0,0 +1,29 @@
+// == let ==
+// Mendeklarasikan dan menginisialisasi variabel 
+let y = 50;
+let benda = "laptop";
+
+// Menampilkan variabel ke dalam console
+console.log(y); // 50
+console.log(benda); // laptop
+
+// Variabel let dapat diinisialisasi ulang nilainya
+benda = "buku";
+
+// Ketika ditampilkan tidak akan error
+console.log(benda); // buku
+
+// == const ==
+// Mendeklarasikan dan menginisialisasi variabel
+const z = 80;
+const minuman = "jus";
+
+// Menampilkan variabel ke dalam console
+console.log(z); // 80
+console.log(minuman); // jus
+
+// Variabel const tidak dapat diinisialisasi ulang nilainya
+minuman = "susu";
```

Comment:
Yang ini nimbulin error di codacy sama codefactor, di comment aja

```suggestion
// minuman = "susu";
```

Reply:
siap